lib/crx.js

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
the header looks weird. Also `crx.test.js` has no header.

Reply:
Looks like other headers in other files.